conda/recipes/cudf/meta.yaml

```diff
@@ -98,6 +98,7 @@ requirements:
     # xref: https://github.com/rapidsai/cudf/issues/12822
     - cuda-nvrtc
     - cuda-python >=12.0,<13.0a0
+    - pynvjitlink
```

Comment:
Once we have a clearer idea on intended compatibility ( https://github.com/rapidsai/pynvjitlink/issues/48 ), we may want to add some version constraints here

This could be done in a separate PR though

Reply:
Yes, this is reasonable. John proposed `pynvjitlink >=0.1.11,<0.2.0a0` offline, which seems appropriate to me.

Reply:
Yeah though let's discuss in the issue and we can do this as follow up (after this PR is merged)